src/spell.sol

```diff
@@ -37,14 +44,140 @@ contract TinlakeSpell is Addresses {
+        // risk group: 21 - , APR: 10%
+        navFeed.file("riskGroup", 21, ONE, ONE, uint(1000000003170980000), 99.64*10**25);
+        // risk group: 22 - , APR: 10.25%
+        navFeed.file("riskGroup", 22, ONE, ONE, uint(1000000003250250000), 99.62*10**25);
+        // risk group: 23 - , APR: 10.5%
+        navFeed.file("riskGroup", 23, ONE, ONE, uint(1000000003329530000), 99.61*10**25);
+        // risk group: 24 - , APR: 10.75%
+        navFeed.file("riskGroup", 24, ONE, ONE, uint(1000000003408800000), 99.59*10**25);
+        // risk group: 25 - , APR: 11%
+        navFeed.file("riskGroup", 25, ONE, ONE, uint(1000000003488080000), 99.58*10**25);
+        // risk group: 26 - , APR: 11.25%
+        navFeed.file("riskGroup", 26, ONE, ONE, uint(1000000003567350000), 99.56*10**25);
+        // risk group: 27 - , APR: 11.5%
+        navFeed.file("riskGroup", 27, ONE, ONE, uint(1000000003646630000), 99.54*10**25);
+        // risk group: 28 - , APR: 11.75%
+        navFeed.file("riskGroup", 28, ONE, ONE, uint(1000000003725900000), 99.53*10**25);
+        // risk group: 29 - , APR: 12%
+        navFeed.file("riskGroup", 29, ONE, ONE, uint(1000000003805180000), 99.51*10**25);
+        // risk group: 30 - , APR: 12.25%
+        navFeed.file("riskGroup", 30, ONE, ONE, uint(1000000003884450000), 99.5*10**25);
+        // risk group: 31 - , APR: 12.5%
+        navFeed.file("riskGroup", 31, ONE, ONE, uint(1000000003963720000), 99.48*10**25);
+        // risk group: 32 - , APR: 12.75%
+        navFeed.file("riskGroup", 32, ONE, ONE, uint(1000000004043000000), 99.47*10**25);
+        // risk group: 33 - , APR: 13%
+        navFeed.file("riskGroup", 33, ONE, ONE, uint(1000000004122270000), 99.45*10**25);
+    }
+
+    function updateFF1() internal {
+        TinlakeRootLike root = TinlakeRootLike(address(FF1_ROOT_CONTRACT));
+        NAVFeedLike navFeed = NAVFeedLike(address(FF1_FEED));
+
+        root.relyContract(FF1_ASSESSOR, address(this));
+        root.relyContract(FF1_FEED, address(this));
+
+        FileLike(FF1_ASSESSOR).file("seniorInterestRate", 1000000001585489599188229325);
+
+        // risk group: 1 - P A, APR: 5.35%
+        navFeed.file("riskGroup", 1, ONE, ONE, uint(1000000001696470000), 99.5*10**25);
+        // risk group: 2 - P BBB, APR: 5.83%
+        navFeed.file("riskGroup", 2, ONE, ONE, uint(1000000001848680000), 99.5*10**25);
+        // risk group: 3 - P BB, APR: 6.3%
+        navFeed.file("riskGroup", 3, ONE, ONE, uint(1000000001997720000), 99.5*10**25);
+        // risk group: 4 - P B, APR: 6.77%
+        navFeed.file("riskGroup", 4, ONE, ONE, uint(1000000002146750000), 99.5*10**25);
+        // risk group: 5 - C, APR: 13.98%
+        navFeed.file("riskGroup", 5, ONE, ONE, uint(1000000004433030000), 98.5*10**25);
+    }
+
+    function updateDBF1() internal {
+        TinlakeRootLike root = TinlakeRootLike(address(DBF1_ROOT_CONTRACT));
+        root.relyContract(DBF1_FEED, dbf1_oracle);
+    }
```

Comment:
I think we need to deny the old oracle contract, except we need to keep using it as well.
applies to all the oracle swaps